x-pack/legacy/plugins/reporting/server/core.ts

```diff
@@ -96,24 +99,30 @@ export class ReportingCore {
     return (await this.getPluginStartDeps()).enqueueJob;
   }
 
-  public getConfig() {
+  public getLicenseInfo() {
+    return checkLicense(this.getExportTypesRegistry(), this.license);
+  }
```

Comment:
going along with https://github.com/elastic/kibana/pull/66331/files#r431516730, we don't need the addition of line 76 if we centralize all the license stuff here:

```suggestion
  public async getLicenseInfo() {
    const { licensing } = await this.getPluginSetupDeps();
    return await licensing.license$
      .pipe(map(license => checkLicense(this.getExportTypesRegistry(), license))), first()).toPromise()
  }
```